scripts/deploy.js

```diff
@@ -80,7 +80,7 @@ async function run() {
       console.error(`Run with --restart to restart the bot.`);
     }
 
-    await ssh.exec(`docker pull devsnek/js-eval`);
+    await ssh.exec(`docker rmi devsnek/js-eval; docker pull devsnek/js-eval`); // delete it first, to force update
```

Comment:
is there no `-f` on `docker pull`?

Reply:
[no](https://docs.docker.com/engine/reference/commandline/pull/) but I might need to add it to the rm command though